### PR TITLE
:memo: Add graphical library benchmark (EPI-16)

### PR DESCRIPTION
**Pull request by AubaneNourry**
I added a .md file with the performance comparison of three graphical libraries: SFML, SDL2, and Allegro, in order to benchmark which one we will use for our project. The test files used to compare the performances are also included.

**In summary, SDL2 is the best choice for the project.**

_Note: This is the second PR for the issue due to the wrong branch name used in the previous one._

### CODE REVIEW

**Review by 6im0n — APPROVED** [review]
Good for me !

File: documentation/benchmarks/graphical_library_benchmark/graphical_library_benchmark.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+#### In conclusion, **SDL2**'s performance and the ecosystem of external libraries make it the preferred choice for developing a robust 2D game engine with complex features as we need it.
+
+---
+
+## Appendix
+
+### Test Scripts
+
+Include the test scripts and commands used to run the benchmarks.
```
Comment (Marius-P1) [inline documentation/benchmarks/graphical_library_benchmark/graphical_library_benchmark.md]: You can add a note to remind people that they should download each library before compiling

**Review by Marius-P1 — APPROVED** [review]
Just a minor comment, but otherwise the benchmark is excellent.

File: documentation/benchmarks/graphical_library_benchmark/graphical_library_benchmark.md (hunk 1 above)
Comment (AubaneNourry) [reply documentation/benchmarks/graphical_library_benchmark/graphical_library_benchmark.md]: I've added the commands i used to install the librairies

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Review by axel-eck — APPROVED** [review]

**Review by landryarki — APPROVED** [review]

**Pull request merged**